tests/repository_test.py

```diff
@@ -79,6 +79,14 @@ def _test_hook_repo(
     assert _norm_out(out) == expected
 
 
+def test_conda_hook(tempdir_factory, store):
+    _test_hook_repo(
+        tempdir_factory, store, 'conda_hooks_repo',
```

Comment:
did you forget to check this one in?

Reply:
Yes, fixed. We probably need to install conda in the CI.